cmd/update-notifier.go

```diff
@@ -27,6 +27,58 @@ import (
 	"github.com/fatih/color"
 )
 
+// humanizeDuration converts time.Duration to human readable duration.
```

Comment:
Reverting this commit and making it as a fresh PR.

I see the struct `humanizedDuration()` is used only in `colorizeUpdateMessage()` for printing duration and nowhere I see its used.  For that a conversion function is good enough to go than passing around a struct.